dev/acf-export.json

```
@@ -2337,6 +2337,44 @@
                 "width": "",
                 "height": ""
             },
+            {
+                "key": "field_5d543d9c27f4c",
+                "label": "Degree Video Caption",
+                "name": "degree_video_caption",
+                "type": "text",
+                "instructions": "Text displayed under the video on degrees",
```

These instructions will be viewed on each degree, so we could edit them to follow in the format of some of the instructions above for other fields. For this field, we could say something like:
```suggestion
                "instructions": "Enter a caption that describes the chosen degree video.",
```